javascript.js

```diff
@@ -1,19 +1,19 @@
-if (window.location.href=="https://accio1.github.io/SSE/") {
-	window.location.replace("https://accio1.github.io/SSE/home/");
-	}
+if (window.location.hash === "" || window.location.hash === "#") {
+	window.location.href = "/home";
```

Comment:
Please change this to window.location.replace, as that makes the user not able to use the back arrow key to go back to the incorrect page.